storage/remote/client.go

```diff
@@ -145,8 +145,11 @@ func NewWriteClient(name string, conf *ClientConfig) (WriteClient, error) {
 	}, nil
 }
 
+const defaultBackoff = 0
```

Comment:
This probably doesn't need to be a constant since 0 is the default value anyway. I can see an argument for readability so I don't feel strongly.

Reply:
Yeah, I feel that would be helpful for readability especially for `retryAfterDuration()` func.